src/components/profile/UserEmailForm/EmailAdd/index.jsx

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import { Container } from "react-bootstrap";
+import { useForm } from "react-hook-form";
+import Joi from "joi";
+import { joiResolver } from "@hookform/resolvers/joi";
+import InputErrorMessage from "../../../InputErrorMessage";
+import { getErrorClass } from "../../../../utils/formErrorHelpers";
+
+const schema = Joi.object({
+  email: Joi.string().required().email({ tlds: {} }).messages({
+    "string.empty": `Please provide your email`,
+    "string.email": `Please provide a valid email`,
+  }),
+});
+
+export default function EmailAdd({ addNewEmail }) {
+  const { register, getValues, errors, reset, trigger } = useForm({
+    mode: "onBlur",
+    resolver: joiResolver(schema),
+  });
+
+  const handleClick = async () => {
+    console.log(errors);
+    const isValid = await trigger("email");
+    if (isValid) {
+      const email = getValues("email");
+      addNewEmail(email);
+      reset();
+    }
+  };
+
+  return (
+    <Container fluid className="emailBlock">
+      <div className="emailAddress">
+        <form>
+          <div className="formInput">
+            <input
```

Comment:
i'm confused. in a separate PR you made an Input component. why isn't that being reused everywhere?

Reply:
just waiting to merge the previous PR to use it everywhere : )

Reply:
nope. use it here, use git merge.